README.md

```diff
@@ -238,4 +238,47 @@ How to check status and read connection metrics:
     curl --cacert test-keys/root.crt https://localhost:6060/_metrics
```

Comment:
I think this is just the PR view getting confused -- the current readme has that and isn't italicized